README.md

```diff
@@ -0,0 +1,197 @@
+received. As the expected use is in a master-slave communication, IRQ doesn't
+have to be handled outside ModbusPort subVI. Data are writen into receiving
+FIFO.
+
+CRC16 isn't checked on received FIFO and the CRC16 (if part of the
+communication) is stored into receiving FIFO.
+
+# Instructions
+
+Opcodes 1-8 are used for transmission (Tx FIFO). Instructions 9 and 10 are used
+in received response (Rx FIFO).
+
+ OpCode | Description
+ ------ | -----------
+ 0x1    | sends data. Data are in bits 9-0, and includes start and stop bits. The least significant bit (bit 0) is a start bit, must be 0. 8 data bits follow. Bit 9 is stop  bit, must be 1. To write 8 bit data, shift left (with 0 fill) by 1, or result with 0x0200
+ 0x2    | End of frame, delay in ticks (data). Should be 0xda, resulting in 0x20da written
+ 0x3    | delay for data ticks
+ 0x4    | delay for data microseconds
+ 0x5    | delay for data milliseconds
+ 0x6    | wait for a received frame. Data = timeout in us (microseconds, 1/1M of a second, 40 ticks). Recommended value is 0x3e8 = 1ms.
+ 0x7    | triggers IRQ
+ 0x8    | wait for trigger (ModbusTrigger == true)
+ 0x9    | received data (on received FIFO)
+ 0xA    | received end of frame (on received FIFO)
+
+For synchronization with the other modbus ports, a frame should be prefixed
+with 0x8 command. Data wouldn't be transmitted before ModbusTrigger == true.
+ModbusTrigger shall be turned to false by process/subVi calling 0x8 command.
+
+## Example
+
+As an example, to call Modbus function 17 = 0x11 for address 0x81, so the
+payload is:
+
+```
+0x81 0x11
+```
+
+| Serb. | Func. | CRC   |
+| Addr  | Code  |       |
+| ----- | ----- | ----- |
+| 81    | 11    | A1 EC |
+
+*[CRC16/MODBUS](https://crccalc.com) of 0x81 0x11 = 0xeca1, transmitted as low
+endian (hence 0xa1 0xec, right shifted by 1 = 0x142 0x1d8; with stop bit =
+0x342 0x3d8).*
+
+the following has to be commanded:
+
+* set WaitForTrigger
+* write payload and CRC
+* wait 1ms for reply
+* signal TriggerIRQ
+
+For that, the following 2 bytes numbers (in hex) shall be written into Tx FIFO:
+
+```
+0x8000 0x1302 0x1222 0x1342 0x13d8 0x20da 0x63e8 0x7000
+```
+
+Assuming the device returns the following data:
+
+| Serb. | Func. | Length | Unique ID         | ILC     | Net. | ILC      | Net.Node | Maj. | Min. | Firmware    | CRC   |
+| Addr  | Code  |        |                   | AppType | Node | Sel.Opt. | OPtions  | Rev. | Rev. | Name        |       |
+| ----- | ----- | ------ | ----------------- | ------  | ---- | -------- | -------- | ---- | ---- | ----------- | ----- |
+| 81    | 11    |   10   | 12 34 56 78 90 AA |   FF    |   BB |   CC     |   DD     |   EE |   11 | 53 74 61 72 | A7 9F |
+
+this shall be read from receiving FIFO:
+
+```
+0x9302 0x9222 0x9220 0x9224 0x9268 0x92ac 0x92f0 0x9320 0x9354 0x93fe 0x9398 0x93ba 0x93dc 0x9222 0x92a6 0x92e8 0x92c2 0x92e4 0x934e 0x933e 0xA000
+```
+
+**It's reader's responsibility to check CRC16.**
```

Comment:
I'm still unclear on whether CRC16 is required or you are simply suggesting it.

Reply:
Changed to:

:warning: It is reader's responsibility to check CRC16. No checking is done inside the library.

CRC should be checked - otherwise disasters will follow.